src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -296,7 +295,7 @@
   [{:keys [last-in-group?
            identicon
            from in-popover? timestamp-str
-           deleted-for-me?]
+           deleted-for-me? pinned]
```

Comment:
if it's a boolean, consider using `pinned?`

Reply:
That's a key extracted from the message object map